docs/user/README.md

```diff
@@ -23,6 +23,7 @@ To support telemetry for your applications, Kyma's Telemetry module provides the
 - Tooling for collection, filtering, and shipment: Based on the [Open Telemetry Collector](https://opentelemetry.io/docs/collector/) and [Fluent Bit](https://fluentbit.io/), you can configure basic pipelines to filter and ship telemetry data.
 - Integration in a vendor-neutral way to a vendor-specific observability system (traces and metrics only): Based on the [OpenTelemetry protocol (OTLP)](https://opentelemetry.io/docs/reference/specification/protocol/), you can integrate backend systems.
 - Guidance for the instrumentation (traces and metrics only): Based on [Open Telemetry](https://opentelemetry.io/), you get community samples on how to instrument your code using the [Open Telemetry SDKs](https://opentelemetry.io/docs/instrumentation/) in nearly every programming language.
+- Enriching telemetry data by adding common attributes (traces and metrics only). This is done in compliance with established semantic conventions, ensuring that the enriched data adheres to industry best practices and is more meaningful for analysis.
```

Comment:
```suggestion
- [Enriching](#automatic-telemetry-enrichment) telemetry data by automatically adding common attributes (traces and metrics only). This is done in compliance with established semantic conventions, ensuring that the enriched data adheres to industry best practices and is more meaningful for analysis.
```

We can link to that section.
And I'm not sure whether it was by purpose that you didn't mention that it's automatic? If by purpose, drop the "automatically" from my suggestion :)

Reply:
It is indeed automatic, the user does not have to do anything to get telemetry data enriched